dali/pipeline/executor/executor_test.cc

```diff
@@ -47,20 +48,24 @@ class ExecutorTest : public GenericDecoderTest<RGB> {
     return spec;
   }
 
-  inline void PruneGraph(Executor *exe) const {
+  inline void PruneGraph(ExecutorBase *exe) const {
     exe->PruneUnusedGraphNodes();
   }
 
-  vector<HostWorkspace> CPUData(Executor *exe, int idx) const {
-    return std::get<static_cast<int>(OpType::CPU)>(exe->wss_[idx].op_data);
+  // TODO(klecki): adjust to refactored code
+  vector<HostWorkspace> CPUData(ExecutorBase *exe, int idx) const {
+    // return std::get<static_cast<int>(OpType::CPU)>(exe->wss_[idx].op_data);
```

Comment:
?

Reply:
This is used in test that is currently disabled, I will try to revive it later but it is tied to internal representation of old implementation instead of behaviour of executor.